src/plugins/intel_gna/src/gna_plugin_query_api.cpp

```diff
@@ -60,6 +61,21 @@ Parameter GNAPlugin::GetMetric(const std::string& name, const std::map<std::stri
         return GNADeviceHelper::GetGnaLibraryVersion();
     } else if (ov::execution_devices == name) {
         return decltype(ov::execution_devices)::value_type {GetName()};
+    } else if (name == ov::caching_properties) {
+        std::vector<ov::PropertyName> cachingProperties;
+        cachingProperties.push_back(ov::PropertyName(ov::intel_gna::scale_factors_per_input.name(), ov::PropertyMutability::RO));
+        cachingProperties.push_back(ov::PropertyName(ov::intel_gna::firmware_model_image_path.name(), ov::PropertyMutability::RO));
+        cachingProperties.push_back(ov::PropertyName(ov::intel_gna::execution_mode.name(), ov::PropertyMutability::RO));
```

Comment:
Are you sure that `execution_mode` is RO after compilation? Currently it can be modified for compiled model. Actually it is possible to run compiled model using HW or using SW mode.

Reply:
You are correct, I will remove this property from the list.